### PR TITLE
Adding Yoast Seo helper

### PR DESCRIPTION
**Pull request by iruzevic**
Related to this issue https://github.com/infinum/eightshift-frontend-libs/issues/231

I have adde an script that you should just call in your `application-block-editor.js` like this:

```js
import { yoastSeo } from '@eightshift/frontend-libs/scripts/editor';

yoastSeo();
```

This helper will search all blocks and components manifests and find attributes that have `"seo": "true"` key.
This key will be added in to the content and proceed by the YoastSeo Analysis plugin.

### CODE REVIEW

File: scripts/editor/yoast-seo.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+			}
+		});
+
+		return output;
+	}
+
+	// Find all strings using regex in the provided data set of dynamic attributes.
+	findStrings(key, data) {
+
+		console.log(key);
```
Comment (kmaric69) [inline scripts/editor/yoast-seo.js:94]: remove console.log

File: scripts/editor/yoast-seo.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+		return output;
+	}
+
+	// Find all menifests in the provided require.context.
+	findManifests(items) {
+
+		const output = [];
+
+		// Find all attrobutes in that contains
```
Comment (kmaric69) [inline scripts/editor/yoast-seo.js:73]: you have a typo here: attributes

File: scripts/editor/yoast-seo.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+		this.findManifests(dataType).forEach((element) => {
+			output += this.findStrings(element, data).map((item) => {
+				return item;
+			});
+		});
+
+		return output;
+	}
+
+	// Find all menifests in the provided require.context.
```
Comment (kmaric69) [inline scripts/editor/yoast-seo.js:68]: typo: manifest

File: scripts/editor/yoast-seo.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+			let newData = data;
+
+			newData += this.checkData(require.context(`./../../components`, true, /manifest.json$/), newData);
+			newData += this.checkData(require.context(`./../../custom`, true, /manifest.json$/), newData);
+
+			return newData;
+	}
+
+	// Check for the menifest data and append.
```
Comment (kmaric69) [inline scripts/editor/yoast-seo.js:54]: typo

Comment (gabriel-glo) [reply scripts/editor/yoast-seo.js:54]:
```suggestion
	// Check for the manifest data and append.
```

File: scripts/editor/yoast-seo.js (hunk 3 above)
Comment (gabriel-glo) [reply scripts/editor/yoast-seo.js:68]:
```suggestion
	// Find all manifests in the provided require.context.
```

File: scripts/editor/yoast-seo.js (hunk 2 above)
Comment (gabriel-glo) [reply scripts/editor/yoast-seo.js:73]:
```suggestion
		// Find all attributes in that contains 
```

File: scripts/editor/yoast-seo.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+		}
+
+		return output;
+	}
+}
+
+export const yoastSeo = () => {
+	domReady(() => {
+		/**
+		* Adds eventlistener to load the plugin.
```
Comment (gabriel-glo) [inline scripts/editor/yoast-seo.js:123]:
```suggestion
		* Adds event listener to load the plugin.
```

File: scripts/editor/yoast-seo.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+		*/
+		if ( typeof YoastSEO !== 'undefined' && typeof YoastSEO.app !== 'undefined' ) {
+		new YoastSEOCustomData();
```
Comment (volfkarlo) [inline scripts/editor/yoast-seo.js:126]: Is tab missing here?

**Comment by iruzevic** [comment]
hahah there is a limit on how manifest can be spelled in one PR :D

**Review by volfkarlo — APPROVED** [review]

**Review by gabriel-glo — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Review by IvanGrginovInf — APPROVED** [review]

**Pull request merged**